frontend/src/modules/member/components/form/member-form-organizations-drawer.vue

```diff
@@ -23,8 +23,12 @@
               </h6>
               <p class="text-2xs leading-5 text-gray-500">
                 {{ organization.memberOrganizations.title }}
-                <span v-if="organization.memberOrganizations.dateStart || organization.memberOrganizations.dateEnd">
-                  ・ {{ formatDate(organization.memberOrganizations.dateStart) }} →
+                <span
+                  v-if="(organization.memberOrganizations.dateStart || organization.memberOrganizations.dateEnd)
+                    && organization.memberOrganizations.title"
+                > ・ </span>
+                <span v-if="(organization.memberOrganizations.dateStart || organization.memberOrganizations.dateEnd)">
+                  {{ formatDate(organization.memberOrganizations.dateStart) }} →
                   {{ formatDate(organization.memberOrganizations.dateEnd) }}</span>
```

Comment:
Shouldn't this display be exactly the same one as we have on `_aside-organizations.vue`?